### DISCUSSION TITLE
Changing font size in vanilla-jsoneditor

### DISCUSSION THREAD

**Discussion by SalamanderXing** [josdejong/svelte-jsoneditor#176] [Q&A] [answered]
Hi, I'm having trouble changing the font size in vanilla-jsoneditor (I'm using React). I have seen you have an example, but only for the svelte version.
I have tried something like this in my CSS:
```
:root, * {
	--jse-font-size: 20px !important;
	--jse-font-size-mono: 20px !important;
}
```
But it gets overwritten. How should I do this?

**Comment by josdejong** [comment, marked as answer]
I've added an example here: https://github.com/josdejong/svelte-jsoneditor/blob/main/examples/browser/custom_theme_color.html

You can define the `--jse-font-size-mono` variable for example on the DIV container where you're going to load the JSONEditor so it will override the default defined on `:root`. Alternatively, you can use `:root {...}` but then you will have to make sure your css is loaded _after_ the css of the JSONEditor so it will override the default.

**Reply by SalamanderXing** [reply]
Thank you!
